pkg/test/integration/common/framework.go

```diff
@@ -44,14 +44,13 @@ import (
 
 const (
 	dwdIgnoreScalingAnnotation = "dependency-watchdog.gardener.cloud/ignore-scaling"
-
-	// Suffix for the`kubernetes-io-cluster` tag and cluster name for the orphan resource tracker
-	targetClusterPlaceholder = "integration-test-cluster"
 )
 
 var (
 	// path for storing log files (mcm & mc processes)
 	targetDir = filepath.Join("..", "..", "..", ".ci", "controllers-test", "logs")
+	// Suffix for the`kubernetes-io-cluster` tag and cluster name for the orphan resource tracker
+	targetClusterName = os.Getenv("TARGET_RESOURCE_GROUP")
```

Comment:
What happens if the env var is not provided? is there a default value?

Reply:
This needs to be provided as we can't have a deafult value for the target cluster

Reply:
yes, but if it's not compulsory for other cloud providers then does the absence of this env var not impact anything?

Reply:
no because it is not even used in other cloudprovider code , so no impact whether it's present or absent.

Reply:
it might impact some tags, but testing wise other providers don't need it so tests are running fine.

Reply:
Okay, fair enough then